bin/setup

```diff
@@ -17,8 +17,8 @@ FileUtils.chdir APP_ROOT do
   system! "gem install bundler --conservative"
   system("bundle check") || system!("bundle install")
 
-  # Install JavaScript dependencies if using Yarn
-  # system('bin/yarn')
+  # Install JavaScript dependencies
+  system("yarn")
```

Comment:
We do have a `bin/yarn` script, so why not use that as per the template? (although I'm not sure it really provides any benefit)

Reply:
Ah, the next commit provides a clue. It's not quite as quick.

Reply:
Oh, I didn't actually see that we had `bin/yarn`. I'll revise this commit.